rotkehlchen/chain/arbitrum_one/constants.py

```diff
@@ -0,0 +1,25 @@
+from rotkehlchen.chain.evm.types import NodeName, WeightedNode, string_to_evm_address
+from rotkehlchen.constants.misc import ONE
+from rotkehlchen.fval import FVal
+from rotkehlchen.types import SupportedBlockchain, Timestamp, deserialize_evm_tx_hash
+
+ARBITRUM_ONE_ETHERSCAN_NODE_NAME = 'arbitrum one etherscan'
+ARBITRUM_ONE_GENESIS = Timestamp(1622240000)
+ARBITRUM_ONE_ETHERSCAN_NODE = WeightedNode(
+    node_info=NodeName(
+        name=ARBITRUM_ONE_ETHERSCAN_NODE_NAME,
+        endpoint='',
+        owned=False,
+        blockchain=SupportedBlockchain.ARBITRUM_ONE,
+    ),
+    weight=ONE,
+    active=True,
+)
+
+CPT_ARBITRUM_ONE = 'arbitrum_one'
+
+ARCHIVE_NODE_CHECK_ADDRESS = string_to_evm_address('0x1c7d91ccBdBf378bAC0F074678b09CB589184e4E')
+ARCHIVE_NODE_CHECK_BLOCK = 42
+ARCHIVE_NODE_CHECK_EXPECTED_BALANCE = FVal('1.499680041387004733')
```

Comment:
I used web3.eth.get_balance to confirm that balance. But I can change it to that address since it's easier to verify. By the way is there a way to find such addresses or you just check adresses of first transactions one by one?

Reply:
no it's unfortunately quite manual a process to find out a good address. But with a good intuition of how it works you can do it fast.

I just check which transaction is closer to genesis and touches an address easily and has very few history events.